hyper/http11/response.py

```diff
@@ -48,7 +49,8 @@ def __init__(self, code, reason, headers, sock, connection=None):
         # bother checking for content-length, we just keep reading until
         # we no longer can.
         self._expect_close = False
-        if b'close' in self.headers.get(b'connection', []):
+        if (b'close' in self.headers.get(b'connection', []) or
+                b'close' in self.headers.get(b'proxy-connection', [])):
             self._expect_close = True
```

Comment:
Okay 😃
But there's a problem: `HTTP/1.0 200 Connection established` response to CONNECT will be considered as close-delimited, so the Response class will eat up the tunnel thinking that this is a body of this response.

I will think about this problem in the coming days. 🙂